docs/docs/components/components-multichannel.md

```diff
@@ -26,52 +39,28 @@ As an example, it can be used to adapt native representations of a Text with But
                       
 
 
-## Example
+### Example
 
-Add the following code in your `action` files, under the `render` section.
+As an output, you would get something like this: 
 
 ```javascript
 <Multichannel
-  firstIndex={'a'}
-  boldIndex={true}
-  carousel={{ indexMode: 'letter' }}
-  text={{ indexMode: 'letter' }}
-  indexSeparator={'.'}
-  messageSeparator={'\n\n'}
->
-```
-
-### Description
-
-- `firstIndex` = The index is the option title of a choice. Ex: a. b. or 1. 2. etc.
-
-- `boldIndex`: Applies a bold format to the index element. True or False.
-
-- `Carousel`: Adds a letter or number to the index for a carousel.
-
-- `Text` : Adds a letter or number to the index for a text.
-
-- `Indexseparator`: Adds a dot or dash after the letter or number.
-
-- `Message separator` : Adds a space between the various messages. `{'\n\n'}` adds a line break between the index message and the selection option.
-
-### Output
-
-```javascript
-<Multichannel {...LEGACY_PROPS}>
+        text={{ indexMode: 'number' }}
+        carousel={{ showTitle: true, ... }}
+        indexSeparator='.'
+        messageSeparator={null}
+      >
   <Text>
     Some with buttons
-    {[
       <Button key={'1'} payload='payload1'>
         Button 1
-      </Button>,
+      </Button>
       <Button key={'2'} path='path1'>
         Button 2
-      </Button>,
+      </Button>
       <Button key={'3'} url='http://testurl.com'>
         Visit website
-      </Button>,
-    ]}
+      </Button>
   </Text>
 </Multichannel>
```

Comment:
this is not an example of the output, but what it generates. I would ask for an example where multichannel is used an adapt it in order to show a screenshot of how it looks in for example, Whatsapp.

Reply:
@vanbasten17 I've just added a visual example and modified the code.